index.html

```diff
@@ -4,16 +4,102 @@
 <head>
   <meta charset="utf-8">
   <title>Great Idea - Responsive I</title>
-
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
You may want to bookmark this or copy the snippet, as you'll need it frequently.